src/mygrad/nnet/activations/sigmoid.py

```diff
@@ -7,7 +7,7 @@
 class Sigmoid(Operation):
     def __call__(self, a):
         self.variables = (a,)
-        x = np.asarray(-1.0 * a.data)
```

Comment:
Why did you get rid of this?

Reply:
Whoops, thought `Operation` was getting updated to account for this; added back